autogpt_platform/frontend/src/components/NavBar.tsx

```diff
@@ -43,7 +44,13 @@ export async function NavBar() {
           <SheetContent side="left">
             <nav className="grid gap-6 text-lg font-medium">
               <Link
-                href="/marketplace"
+                href={
+                  getBehaveAs() === "CLOUD"
+                    ? "/marketplace"
+                    : "http://dev-builder.agpt.co/marketplace"
```

Comment:
Don't we already have NEXT_PUBLIC_AGPT_MARKETPLACE_URL ?
Can we just use it and set the default to cloud or local "/marketplace" instead of utilizing a new env.

Reply:
jk we don't want to do this